rps.js

```diff
@@ -0,0 +1,20 @@
+const playGame= function(userChoice){
+   //in Pc random choice i will asgin numbers,
+   // 1 for rock, 2 for paper, and 3 for scissors
+   let pcChoice=Math.floor((Math.random() * 3) + 1); 
+   if(userChoice==="rock"){
```

Comment:
please use if and else if statements.

using only if  will give you bugs in future programs.